tests/robot/resources/keywords.robot

```diff
@@ -177,14 +177,12 @@ Build enclave EDGE URL
 
 Get token from EDI
     [Documentation]  get token from EDI for the EDGE session
-    [Arguments]     ${enclave}   ${model_id}   ${model_version}
```

Comment:
@skayerok why did you delete tests for RESTful API? legionctl is not gonna be a single client for this API. Keep test for RESTful API too

Reply:
Ok, reverted back these tests